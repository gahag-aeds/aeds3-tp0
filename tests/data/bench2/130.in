130 65
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 17 28 9 9 63 44 15 67 67 33 90 37 54 66 2 63 67 77 91 93 60 86 90 8 3 92 27 58 98 39 56 67 67 18 76 82 14 44 49 82 29 91 71 83 57 73 47 25 2 90 70 63 76 61 71 79 5 50 37 56 89 46 23 8 16 52 42 30 48 91 
Add 38 80
Sum 75 117
Add 73 115
Max 11 53
Add 46 88
Sum 21 63
Sub 39 81
Max 83 125
Add 50 92
Sum 30 72
Sub 30 72
Max 38 80
Sub 82 124
Sum 30 72
Add 19 61
Min 13 55
Add 61 103
Max 77 119
Add 55 97
Sum 42 84
Sub 1 43
Max 23 65
Add 17 59
Sum 36 78
Sub 3 45
Min 7 49
Sub 38 80
Sum 45 87
Add 10 52
Min 73 115
Add 52 94
Min 40 82
Add 60 102
Min 37 79
Sub 50 92
Sum 18 60
Sub 49 91
Max 80 122
Add 6 48
Sum 12 54
Add 22 64
Sum 27 69
Sub 41 83
Min 16 58
Sub 57 99
Max 59 101
Sub 20 62
Min 22 64
Add 40 82
Max 74 116
Sub 55 97
Sum 33 75
Add 1 43
Sum 63 105
Sub 60 102
Sum 3 45
Sub 69 111
Min 10 52
Add 47 89
Min 79 121
Sub 82 124
Sum 3 45
Sub 31 73
Min 46 88
Add 39 81
