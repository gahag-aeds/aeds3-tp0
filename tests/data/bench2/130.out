2508
96
1893
99
1771
99
1728
2
100
2071
101
1921
3
2078
2
3
3
1986
101
2036
1901
4
100
3
100
1792
2448
2132
4
1
2132
3
